userprofile/factories.py

```diff
@@ -65,9 +65,12 @@ class Meta:
 
 def make_user_and_organization(organization_name="Test Org", github_handle="test-org", github_username="edx"):
     """
-    Create user, userprofile and organization needed for reference on instances
+    Create user, userprofile and organization needed for reference on instances.
+    We're using the github_username passed to this method to create User and
```

Comment:
I understand that you experienced this case (some test failing because of this), but there's no need to mention it; it doesn't directly help the next person who uses the code. Better explain what the function does, and what is expected from the caller. E.g. „If provided, it also sets the user's GitHub username (which must be unique)“. And if you prefer, we don't even need to say that the github username should be unique (it's like User.username): someone who writes tests will notice if a test tries to reuse an existing github_username and fails.